buildcfg.txt

```diff
@@ -2187,7 +2187,7 @@ CANDA0	8	15
 CBRAA0	14	61
 CHGGA0	-107	-117
 CHGGB0	-107	-121
-CHGFA0	-126	-95
+CHGFA0	-126	-98
```

Comment:
An offset of (0,0) means the upper left corner of the image is drawn on the upper left corner of the screen, and positive values shift the image's anchor spot downwards and to the right (thus moving the image position up and to the left).

A lot of X values are around the 100-120 range since that's half of 320 less half the width of a typical centered weapon sprite.